### PR TITLE
Nend amp-ad support.

### PR DESCRIPTION
**Pull request by nend-net**
Could you merge 'Nend amp-ad support' into ampproject/amphtml master branch?

We have a Google Corporate CLA.
Our CLA: F@N Communications, Inc.
Our company name: F@N Communications, Inc.

Thank you in advance

### CODE REVIEW

File: ads/nend.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * limitations under the License.
+ */
+
+import {writeScript} from '../3p/3p';
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function nend(global, data) {
+  global.nendParam = data;
```
Comment (dvoytenko) [inline ads/nend.js]: Could you please check the permitted values using `checkData` from the `3p` module?

Is `data-nend_params` required? If so - could you please verify its existence via `validateDataExists` in the same module?

**Comment by dvoytenko** [comment]
@nend-net This looks great. Just a couple small comments. PTAL.

**Comment by nend-net** [comment]
@dvoytenko 
Thank you for your advice.　
I modified the source and pushed to ’nend-net:master' branch a minutes ago. 
Could you check my commit again?

**Comment by dvoytenko** [comment]
@nend-net Thanks! Merging.

**Comment by nend-net** [comment]
@dvoytenko Thanks a lot!

**Pull request merged**